rolling-shutter/keyperimpl/optimism/bootstrap/bootstrap.go

```diff
@@ -34,6 +34,7 @@ func BootstrapValidators(config *Config) error {
 	}
 
 	ms := fx.NewRPCMessageSender(shmcl, config.SigningKey.Key)
+	ms.AllowedToSend.Store(true)
```

Comment:
same question as in #489 : it is intentional to fix this in the `optimism` keyperimpl?